mitosheet/src/plugin.tsx

```diff
@@ -138,6 +138,8 @@ function activateMitosheetExtension(
             const codeLines = args.code as string[];
             const telemetryEnabled = args.telemetryEnabled as boolean;
             const publicInterfaceVersion = args.publicInterfaceVersion as PublicInterfaceVersion;
+            const triggerDialog = args.triggerDialog as () => void;
+            const overwriteCode = args.overwriteCode as boolean | undefined;
```

Comment:
Let's rename this variable throughout to something better. Feel free to take another shot at a name -- let's be more clear. OverwriteCode when ?

Reply:
`overwriteIfUserEditedCode`?